### PR TITLE
LINK-2059 | Check order and payment from Tapla API

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Check that order and payment exist in the payment completed page.
Re-route payment completed page to `/success` and payment cancelled to `/cancelled`

## Closes
[LINK-2059](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2059)

[LINK-2059]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2059?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr157-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr157-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr157-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr157-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr157-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: next.config.js [hunk 1]
```diff
@@ -8,6 +8,20 @@ const { i18n } = require('./next-i18next.config');
 const moduleExports = {
   i18n,
   reactStrictMode: true,
+  async redirects() {
+    return [
+      {
+        source: '/payment/cancelled',
+        destination: '/cancelled',
```
Comment (harriris-vincit) [inline next.config.js:15]: In the Talpa Experience API repo, the path seems to be `/failure`

**Review by harriris-vincit — APPROVED** [review]
One comment about the failed result path, but once that is resolved, this can be merged.

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr157-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr157-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr157-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr157-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**